api/TextEncoder.json

```diff
@@ -70,86 +82,408 @@
+            "chrome": [
+              {
+                "version_added": "53",
+                "notes": "Does not accept parameters. Supports only <code>uft-8</code> encoding."
+              },
+              {
+                "version_added": "38",
+                "version_removed": "53",
+                "notes": "Throws <code>RangeError</code> exception for unknown encoding type"
+              }
+            ],
+            "chrome_android": {
+              "version_added": "38"
+            },
+            "edge": {
               "version_added": null
             },
-            "chrome": {
+            "edge_mobile": {
               "version_added": null
             },
-            "chrome_android": {
+            "firefox": [
+              {
+                "version_added": "48",
+                "notes": "Does not accept parameters. Supports only <code>uft-8</code> encoding."
+              },
+              {
+                "version_added": "38",
+                "version_removed": "48",
+                "notes": "Invalid parameter leads to the standard <code>RangeError</code> exception."
+              },
+              {
+                "version_added": "19",
+                "version_removed": "38",
+                "notes": "Invalid parameter leads to a <code>TypeError</code> exception."
+              },
+              {
+                "version_added": "18",
+                "notes": "Slightly different version of the spec"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "48",
+                "notes": "Does not accept parameters. Supports only <code>uft-8</code> encoding."
+              },
+              {
+                "version_added": "38",
+                "version_removed": "48",
+                "notes": "Invalid parameter leads to the standard <code>RangeError</code> exception."
+              },
+              {
+                "version_added": "19",
+                "version_removed": "38",
+                "notes": "Invalid parameter leads to a <code>TypeError</code> exception."
+              },
+              {
+                "version_added": "18",
+                "notes": "Slightly different version of the spec"
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "25"
+            },
+            "opera_android": {
               "version_added": null
             },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "worker_support": {
```

Comment:
I think, to be consistent with other areas in BCD, we can have a single top-level workers support feature and not have duplicates for every subfeature.